Networking/Networking/Model/Product/ProductCategory.swift

```diff
@@ -58,3 +73,21 @@ extension ProductCategory: Comparable {
             (lhs.categoryID == rhs.categoryID && lhs.name == rhs.name && lhs.slug < rhs.slug)
     }
 }
+
+// MARK: - Update functions
+//
+extension ProductCategory {
+    public func updateWith(parentID: Int64) -> ProductCategory {
```

Comment:
This would probably be deprecated when we have that `copy()` pattern. 😄 For now, would you mind following the method name convention used in [`ProductUpdater`](https://github.com/woocommerce/woocommerce-ios/blob/develop/Yosemite/Yosemite/Model/Updaters/ProductUpdater.swift)?

```suggestion
    public func parentIDUpdated(parentID: Int64) -> ProductCategory {
```

The `updateWith()` doesn't sound like a new instance will be created. 😐

Reply:
Fixed https://github.com/woocommerce/woocommerce-ios/pull/2079/commits/dadb2f45367f4bf64979fd6c0f972564aa400d23